### DISCUSSION TITLE
Administrator identity issue

### DISCUSSION THREAD

**Discussion by yixuan1912** [rustdesk/rustdesk#7696] [General]
### Bug Description

When opening cmd as an administrator, the authorization interface cannot be displayed, but pause is displayed.

![image](https://github.com/rustdesk/rustdesk/assets/44570514/27b81383-6546-4fa7-a95f-4cd62a9875ce)

### How to Reproduce

When using rustdesk to remotely connect to another win system, open cmd as an administrator

### Expected Behavior

The interface for administrator identity authorization can be successfully displayed and you can click to agree.

### Operating system(s) on local side and remote side

windows10

### RustDesk Version(s) on local side and remote side

1.2.3-1

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/44570514/27b81383-6546-4fa7-a95f-4cd62a9875ce)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/wiki/FAQ#windows-uac

**Reply by yixuan1912** [reply]
I gave rustdesk administrator rights and turned on all permissions
However, I still cannot run cmd as administrator.

![image](https://github.com/rustdesk/rustdesk/assets/44570514/840d07a6-cbe3-4383-9317-99ca51b664b9)

**Reply by yixuan1912** [reply]
I tried it but it doesn't seem to work

![image](https://github.com/rustdesk/rustdesk/assets/44570514/636f637c-6ad7-43c4-bb40-60804bc5a294)
